xml/networking-octavia_admin.xml

```diff
@@ -403,6 +403,148 @@ Updated port: 0b0301b9-4ee8-4fb6-a47c-2690594173f4</screen>
    </para>
   </warning>
  </section>
+ <section xml:id="octavia-admin-delete">
+   <title>Removing load balancers</title>
+   <para>
+     The following instruction details how to delete a load balancer
```

Comment:
proof-reading this again, I think it should be "instructions detail" rather than "instruction details"

Reply:
Or: _The following procedure demonstrates how to delete [...]_

Reply:
Done.